Library/Formula/libgdiplus.rb

```diff
@@ -0,0 +1,32 @@
+class Libgdiplus < Formula
+  desc "C-based implementation of the GDI+ API for Mono"
+  homepage "http://www.mono-project.com/docs/gui/libgdiplus/"
+  url "https://github.com/mono/libgdiplus/tarball/79ae6cdb90163ba5eb601f8691ea9b7558d25371"
+  head "https://github.com/mono/libgdiplus.git", :using => :git, :tag => "master"
```

Comment:
Need nothing after `head "https://github.com/mono/libgdiplus.git"` here. Everything else is the assumed default.